src/Model/Model.php

```diff
@@ -216,4 +216,23 @@ public static function loadFunctionMap(): FunctionMap
     {
         return FunctionMap::loadFunctionMap();
     }
+
+    public function sortPoliciesByPriority(): void
+    {
+        foreach ($this->items['p'] as $ptype => $assertion) {
+            if ($assertion->tokens[0] != sprintf("%s_priority", $ptype)) {
+                continue;
+            }
```

Comment:
@basakest  The first column is not necessarily the priority column, I think `array_search($assertion->tokens, sprintf("%s_priority", $ptype))` can be used here .